### PR TITLE
Update link styles on faq page

### PR DESCRIPTION
**Pull request by hectoruch**
Change faq.json with the underline link class.

### CODE REVIEW

File: app/styles/_base.scss [hunk 1]
```diff
@@ -98,8 +98,28 @@ a {
       top: auto;
     }
   }
+
+  :global &.-underline {
```
Comment (tiagojsag) [inline app/styles/_base.scss]: https://github.com/Vizzuality/GlobalFishingWatch/pull/211/files#diff-da132c0d50c9cd8406b6391530936e2bR84

cant we DRY this?

Comment (andresgnlez) [inline app/styles/_base.scss]: @hectoruch did you make it global because of `faq.json`?

Comment (hectoruch) [inline app/styles/_base.scss]: I just :global because on json file can not add "className". That way we can use -underline class for two ways.

**Comment by hectoruch** [comment]
I think that is ready to merge @tiagojsag

File: app/styles/_base.scss (hunk 1 above)
Comment (tiagojsag) [inline app/styles/_base.scss]: @andresgnlez there's no way here to NOT have the same 20 lines duplicated just to have the "global"?

Comment (hectoruch) [inline app/styles/_base.scss]: @tiagojsag  I was talking with Raul. And is the only way to do that.

**Pull request merged**